### PR TITLE
Fix missing artifact link

### PR DESCRIPTION
**Pull request by sandro-elsweijer**

### CODE REVIEW

File: content/pages/download.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h3 class="widget-title">
 			    Dual licensing
                     </h3><!-- /.widget-title -->
                     <p>
-		    If you want to use t8code under a different licence than GPL, please <a href="{{ SITEURL }}/pages/contact.html">Contact us</a> to discuss possible dual licenses.
+		    If you want to use t8code under a different licence than GPL, ple to discuss possible dual licenses.
```
Comment (Davknapp) [inline content/pages/download.html:51]: This change does not look like an improvement to me. Can you explain your intentions?

Comment (sandro-elsweijer) [reply content/pages/download.html:51]: This was just a test. The link I removed produced an error message while building the website.
I thought that the CI process maybe stops due to this error and therefore the artifact link would not get added.

Comment (sandro-elsweijer) [reply content/pages/download.html:51]: Its just a draft PR, there is no review required here ;)

**Comment by sandro-elsweijer** [comment]
Fixed in #38

**Pull request closed without merging**